security-team/bug-report.md

```diff
@@ -0,0 +1,17 @@
+<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!
```

Comment:
why have bug-report here

Reply:
Because I think that since we have a security vulnerability submission specification, we should also need a template for the submission content

Reply:
bug report is a separate issue, no need to couple with vulnerability. It can be linked in developer guide